tests/2017-03-01-preview/Microsoft.Insights.ActivityLogAlerts.tests.json

```diff
@@ -0,0 +1,76 @@
+{
+  "tests": [
+    {
+      "name": "Activity Log Alert - Administrative",
+      "definition": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActivityLogAlert.json#/resourceDefinitions/activityLogAlerts",
+      "json": {
+        "type": "Microsoft.Insights/activityLogAlerts",
+        "apiVersion": "2017-03-01-preview",
+        "location": "Global",
+        "name": "TEST Administrative Activity Log Alert",
+        "properties": {
+            "scopes": [
+                "/subscriptions/52c65f65-0518-4d37-9719-7dbbfc68c57a"
+            ],
+            "description": "This is a silly description",
+            "enabled": true,
+            "condition": {
+              "allOf": [
+                {
+                  "field": "category",
+                  "equals": "Administrative"
+                },
+                {
+                  "field": "resourceGroup",
+                  "equals": "TEST-RG"
+                },
+                {
+                  "field": "caller",
+                  "equals": "devops@contoso.com"
+                }
+              ]
+            },
+            "actions": {
+                "actionGroups": [
+                {
+                    "actionGroupId": "/subscriptions/52c65f65-0518-4d37-9719-7dbbfc68c57a/resourceGroups/CONTOSO-TEST/providers/Microsoft.Insights/actionGroups/ContosoIncidentActions"
+                }
+                ]
+            }
+        }
+      }
+    },
+    {
+      "name": "Activity Log Alert - Service Health",
+      "definition": "http://schema.management.azure.com/schemas/2017-03-01-preview/Microsoft.Insights.ActivityLogAlert.json#/resourceDefinitions/activityLogAlerts",
+      "json": {
+        "type": "Microsoft.Insights/activityLogAlerts",
+        "apiVersion": "2017-03-01-preview",
+        "location": "Global",
+        "name": "TEST ServiceHealth Activity Log Alert",
+        "properties": {
+            "scopes": [
```

Comment:
Please format this file and use consistent indentation. We always use 2 spaces for indent.